extensions/ui/org.eclipse.smarthome.ui.basic/snippets/main.html

```diff
@@ -61,7 +61,7 @@
 			</div>
 		</script>
 	</head>
-	<body class="mdl-color--grey-100 mdl-color-text--grey-700" data-sitemap="%sitemap%" data-page-id="%id%" data-icon-type="%icon_type%">
+	<body class="mdl-color-text--grey-700" data-sitemap="%sitemap%" data-page-id="%id%" data-icon-type="%icon_type%">

```

Comment:
These classes may need to be left as is for browsers without CSS variables support — BasicUI still works with IE 11 and not-so-recent Android versions. This will introduce a graceful fallback.

https://caniuse.com/#feat=css-variables

Reply:
CSS Custom Properties will fallback gracefully as well if you pass a second argument to them, e.g
```css
background: var(--body-bg, #f5f5f5);
```

And this is what I did in my follow-up commit.